src/modules/keyboardmanager/ui/KeyDropDownControl.cpp

```diff
@@ -376,9 +375,8 @@ void KeyDropDownControl::AddShortcutToControl(Shortcut shortcut, Grid table, Sta
     parent.Children().Clear();
     // Remove references to the old drop down objects to destroy them
     keyDropDownControlObjects.clear();
-
+    
```

Comment:
[nit] formatting, Ctrl+K D can be used on VS to fix this everywhere without manually searching for it